webpack.common.js

```diff
@@ -10,9 +14,18 @@ module.exports = {
         ]
     },
     plugins: [
+        new CleanWebpackPlugin(),
         new CopyPlugin([{
             from: "**/*",
             to: "."
-        }])
+        }]),
+        new CopyPlugin(
+            Assets.map(asset => {
+                return {
+                    from: path.resolve(__dirname, `./node_modules/${asset}`),
+                    to: path.resolve(__dirname, './dist/bower_components')
```

Comment:
I would prefer this directory be named something else to differentiate between resources that are still in `bower_components`.

Reply:
How about `dist/libraries` for things we need copied directly?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/549/commits/af2b49ac303d36de2c107a0eeb98968679609cce